src/tests/issues.targets

```diff
@@ -281,7 +269,7 @@
             <Issue>Depends on implicit tailcalls to be performed</Issue>
         </ExcludeList>
         <ExcludeList Include="$(XunitTestBinBase)/JIT/Regression/JitBlue/DevDiv_255294/DevDiv_255294/*">
-            <Issue>https://github.com/dotnet/runtime/issues/8034 The test causes OutOfMemory exception in crossgen mode.</Issue>
+            <Issue>https://github.com/dotnet/runtime/issues/44341 The test causes OutOfMemory exception in crossgen mode.</Issue>
```

Comment:
Is this the right issue number? #44341 is closed as fixed.

Reply:
Does the test still fail?

Reply:
Ah, ok... I suppose linking to either issue is fine, though #44341 was about a timeout, and #8034 an OOM.